test/unit/spec/localparticipant.js

```diff
@@ -122,19 +122,16 @@ describe('LocalParticipant', () => {
     'unpublishTracks',
   ].forEach(method => {
     describe(`#${method}`, () => {
-      const trackMethod = method.slice(0, -1);
+      const trackMethod = method.match('(add|publish).*') ? 'publishTrack' : 'unpublishTrack';
```

Comment:
Is this change necessary? Also, I see that "addTracks" and "removeTracks" now map to "publishTrack" and "unpublishTrack", rather than "addTrack" and "removeTrack".

Reply:
It is necessary so that we mock the publishTrack and unpublishTrack.